src/main/java/com/google/api/codegen/config/GapicMethodConfig.java

```diff
@@ -272,6 +272,18 @@ static GapicMethodConfig createMethodConfig(
     }
   }
 
+  private static List<String> findHeaderRequestParams(
```

Comment:
This  method returns set converted to list. Maybe it should just return set? Or maybe just a `Collection` is good enough?

Reply:
I want to guarantee method callers that the Collection object has a defined order.